### PR TITLE
Release/v5

### PR DESCRIPTION
**Pull request by kirsty-hames**
- naming refactor
- add icon div to homeButton for consistency with other icon buttons
- bower bump

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,17 +4,15 @@
         "type": "git",
         "url": "git://github.com/cgkineo/adapt-homeButton.git"
     },
-    "framework": ">=2.0.18",
-    "version": "0.0.8",
```
Comment (tomgreenfield) [inline bower.json:8]: Versions have been accidentally deleted. Suggest:
```json
"version": "0.1.0",
"framework": ">=5"
```

File: js/adapt-homeButton.js [hunk 2]
```diff
@@ -33,41 +33,42 @@ define([
...
             var $homeButton = $('<button>', {
                 attr: {
                     'data-event': 'homeButton'
                 },
-                'class': 'base navigation-home-button icon icon-home',
-                'aria-label': altText
-            });
+                'class': 'btn-icon nav__btn nav__home-btn js-nav-home-btn',
+                'aria-label': altText,
+                'role' : 'link'
```
Comment (tomgreenfield) [inline js/adapt-homeButton.js]: Don't need quotes around `role`:
```suggestion
                role: 'link'

**Review by ghost — COMMENTED** [review]
Update indentation (to spaces only) and spacing (2 spaces for everything except json files, which use 4 spaces) for consistency

**Review by moloko — APPROVED** [review]
code 👓 

please could you do a release of the current master branch before merging & releasing this?

**Review by kirsty-hames — COMMENTED** [review]
I can't approve as I opened the PR but I'm happy with the changes thanks 👀

**Review by oliverfoster — APPROVED** [review]

**Comment by moloko** [comment]
> I can't approve as I opened the PR but I'm happy with the changes thanks 👀

Those rules around reviewing really only apply to Adapt OS repos... For our repos, if someone else has done further work on something you started it's absolutely fine for you to review it!

File: example.json [hunk 3]
```diff
@@ -7,14 +9,15 @@
...
-        "_redirectToId": "",
+        "_comment": "Amend co-00 to match the ID of the start / landing page",
+        "_redirectToId": "co-00",
         "alt": "Home"
     }
 
     // contentObjects.json
     "_homeButton": {
-        "_isEnabled": false,
+        "_isEnabled": true,
```
Comment (moloko) [inline example.json:19]: Maybe change `hideBackButton` to `true` for this example?

Comment (guywillis) [reply example.json:19]: I would actually argue that `_hideHomeButton` should be set to `true` and `_hideBackButton` set to `false` to retain the standard Adapt back button functionality. Though that all depends upon the primary use case of the plugin which I'm assuming is to allow users to navigate back to a landing / start page from a menu.

Comment (moloko) [reply example.json:19]: > which I'm assuming is to allow users to navigate back to a landing / start page from a menu.

@guywillis the configuration for that is shown in the previous example

Comment (guywillis) [reply example.json:19]: Amended example as per suggestion

**Comment by guywillis** [comment]
@kirsty-hames Would you mind reviewing my amends please?

**Comment by kirsty-hames** [comment]
review_approval
I can't officially approve but I'm happy with @guywillis's changes thanks  👀

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**